packages/block-editor/src/components/global-styles/advanced-panel.js

```diff
@@ -37,40 +90,61 @@ export default function AdvancedPanel( {
 			}
 		}
 	}
-	function handleOnBlur( event ) {
-		if ( ! event?.target?.value ) {
+	function handleOnBlur( newValue ) {
+		if ( ! newValue ) {
 			setCSSError( null );
 			return;
 		}
-
 		const [ transformed ] = transformStyles(
-			[ { css: event.target.value } ],
+			[ { css: newValue } ],
 			'.editor-styles-wrapper'
 		);
-
 		setCSSError(
 			transformed === null
 				? __( 'There is an error with your CSS structure.' )
 				: null
 		);
 	}
 
+	const [ customCSS, setCustomeCSS ] = useState( '' );
```

Comment:
It think there's a bug where the Additional CSS editor will not show without already having an `inheritedValue`.